lib/checkNewCronJobs.js

```diff
@@ -0,0 +1,121 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const {
+  serverJobAdmins
+} = require('../userWhitelist.json');
+const {
+  getAllChangedFiles,
+  pingAndAssignUsers
+} = require('./utils');
+const CRON_REGISTRY_FILENAME = 'main_cron.py';
+const CRON_TEST_FILENAME = 'core/controllers/cron_test.py';
+const CRON_JOB_FILE = 'core/controllers/cron.py';
+const WIKI_LINK = 'https://github.com/oppia/oppia/wiki/' +
+ 'Testing-jobs-and-other-features-on-production';
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} changedFiles
+ */
+const isCronFileChanged = (changedFiles) => {
+  let result;
+  changedFiles.some((file) => {
+    if (file.filename === CRON_JOB_FILE) {
+      result = true;
+    }
+  });
+  return result;
+};
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} changedFiles
+ */
+const hasAddedTests = (changedFiles) => {
+  let result;
+  changedFiles.some((file) => {
```

Comment:
You should `return`  the result of `changedFiles.some`. Currently, you are returning undefined.
Do this here and elsewhere.

Reply:
done